draft-ietf-quic-qpack.md

```diff
@@ -115,6 +118,9 @@ state which the peer might not have received yet, the BLOCKING flag MUST be
 set.  If the peer does not yet have the appropriate state, such blocks
 might not be processed on arrival.
 
+An encoder MUST NOT have more header blocks outstanding with the BLOCKING flag
+set than the value of SETTINGS_BLOCKING_HEADER_REFERENCES at any time.
```

Comment:
So what does a decoder do if it has to block more times than it said it was willing to?  You should also note that an encoder might violate this agreement but not be detected in doing so.